tests/test_updater_ng.py

```diff
@@ -316,6 +317,19 @@ def test_missing_targetinfo(self) -> None:
         # Get targetinfo for non-existing file
         self.assertIsNone(self.updater.get_targetinfo("file33.txt"))
 
+    @patch.object(os, "replace", wraps=os.replace)
+    @patch.object(os, "remove", wraps=os.remove)
+    def test_persist_metadata_fails(
+        self, wrapped_remove: MagicMock, wrapped_replace: MagicMock,
```

Comment:
I noticed that `black` doesn't like that I put all of the arguments on one line here.
It requires that I should separate each of them on a new line even though this line is far less than 80 characters.
Honestly, I prefer if I leave it the way instead of unnecessarily using a new line for each argument.
It's possible that that's a new version of black requires that all function arguments that require above 80 characters are placed new line and not allow the above behavior anymore.

Reply:
But if we keep it like that, linter will fail for every following PR and not catch errors from the next commands as well.

Reply:
It's probable that there is a black configuration option for that.
First, I want to hear the others opinions and then if they agree with me I will resereach.

Reply:
black is historically not a very configurable tool so I doubt this config exists -- it's supposed to make choosing a style easy by enforcing one ("There is Only One Style"). This argument splitting behaviour is part of the Black Style (and is documented in https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html) and I don't think we should start working around it.

Reply:
I think I found why black was complaining - it was because I placed a comma after the last argument. 🤦 
When I removed it, it was happy.